go.mod

```diff
@@ -89,6 +89,12 @@ require (
 	gopkg.in/guregu/null.v4 v4.0.0
 )
 
+require (
+	github.com/power-devops/perfstat v0.0.0-20210106213030-5aafc221ea8c // indirect
+	github.com/shirou/gopsutil/v3 v3.22.1
+	github.com/yusufpapurcu/wmi v1.2.2 // indirect
+)
```

Comment:
Are these separated intentionally?

Reply:
OK lets move gopsutil up with the other direct deps, and the other two indirect down below. Once they are in those sets, go mod tidy should leave them there (and sort for you).